app/models/miq_server/worker_management/monitor.rb

```diff
@@ -15,7 +15,7 @@ def monitor_workers
     # Clear the my_server cache so we can detect role and possibly other changes faster
     self.class.my_server_clear_cache
 
-    resync_needed = sync_needed?
+    sync_needed?
```

Comment:
@jrafanie any ideas for a better name for this method now that it's not returning a boolean?

Reply:
maybe `sync_server` to contrast with the `sync_workers` call below?

Reply:
The trouble is it's really sync'ing for worker monitoring.  Maybe `sync_monitor` is more appropriate since all of the work there is about monitoring workers and less about the server itself.